src/o365/spo/commands/page/page-get.ts

```diff
@@ -40,8 +38,10 @@ class SpoPageGetCommand extends SpoCommand {
       pageName += '.aspx';
     }
 
+    let ListItemAllFieldsString: string = this.verbose ? 'ListItemAllFields,' : '';
+
     const requestOptions: any = {
-      url: `${args.options.webUrl}/_api/web/getfilebyserverrelativeurl('${Utils.getServerRelativeSiteUrl(args.options.webUrl)}/SitePages/${encodeURIComponent(pageName)}')?$expand=ListItemAllFields/ClientSideApplicationId,ListItemAllFields/PageLayoutType,ListItemAllFields/CommentsDisabled`,
+      url: `${args.options.webUrl}/_api/web/getfilebyserverrelativeurl('${Utils.getServerRelativeSiteUrl(args.options.webUrl)}/SitePages/${encodeURIComponent(pageName)}')?$expand=${ListItemAllFieldsString}ListItemAllFields/ClientSideApplicationId,ListItemAllFields/PageLayoutType,ListItemAllFields/CommentsDisabled`,
```

Comment:
With this request URL there is no difference between verbose and regular output, so the additional filter from line 41 can be removed